webextensions/manifest/browser_specific_settings.json

```diff
@@ -12,16 +12,10 @@
               "version_added": "15",
               "version_removed": "79"
             },
-            "firefox": [
-              {
-                "version_added": "48"
-              },
-              {
-                "version_added": "42",
-                "alternative_name": "applications",
```

Comment:
`"alternative_name": "applications"` was a dummy entry to allow per-version notes here. I removed it to avoid confusion.